### DISCUSSION TITLE
Delta speed control

### DISCUSSION THREAD

**Discussion by NickiNiclas** [Rem0o/FanControl.Releases#2637] [Ideas]
I would like to see a delta mode that controls fan speed based on the temperature delta of two different temp sensors. 
Example: I want to set the speed of my radiator fans based on how high the water temperature is above the ambient temperature.

**Comment by Rem0o** [comment]
Use a mix sensor, function subtract. Assign that to a fan curve.

![image](https://github.com/user-attachments/assets/e66a3df5-96f0-4aba-8035-17e19197f819)

**Reply by NickiNiclas** [reply]
Oh wow thank you! I should have realized! Awesome! ⭐
